app/workers/publishing_api_links_worker.rb

```diff
@@ -0,0 +1,13 @@
+# Worker for batch sending of links to the publishing api
+class PublishingApiLinksWorker < WorkerBase
+  sidekiq_options queue: "publishing_api"
```

Comment:
Perhaps the `bulk_republishing` queue is better, because it has lower priority and won't interfere with normal publishing:

https://github.com/alphagov/whitehall/blob/master/config/sidekiq.yml#L5